swagger.yaml

```diff
@@ -267,6 +270,36 @@ definitions:
         description: The secret's data section in JSON format
         type: string
 
+  metrics:
+    type: object
+    properties:
+      path:
+        description: Path to use when retrieving metrics
+        type: string
+      port:
+        type: integer
```

Comment:
WDYT of including also `format: int32` which is suitable for the port range?

Reply:
Done! Sorry!